lib/client.js

```diff
@@ -79,7 +79,7 @@ client.prototype.request = function(method, path, params, callback){
 
     if(["POST", "PUT"].indexOf(http_options["method"]) >= 0){
         http_options["headers"] = {
-            "Content-Type": "application/json",
+            "Content-Type": params["contentType"] ? params["contentType"] : "application/json",
```

Comment:
Ha, I was going to send them an e-mail as well. That is ok, I tested the API endpoint, and can at lest confirm that `application/json` does not work.